network-store-server/src/main/java/com/powsybl/network/store/server/QueryCatalog.java

```diff
@@ -19,33 +19,34 @@
  */
 public final class QueryCatalog {
 
-    static final String MINIMAL_VALUE_REQUIREMENT_ERROR = "Function should not be called without at least one value.";
+    public static final String MINIMAL_VALUE_REQUIREMENT_ERROR = "Function should not be called without at least one value.";
 
     static final String VARIANT_ID_COLUMN = "variantId";
     static final String UUID_COLUMN = "uuid";
-    static final String NETWORK_UUID_COLUMN = "networkUuid";
-    static final String VARIANT_NUM_COLUMN = "variantNum";
+    public static final String NETWORK_UUID_COLUMN = "networkUuid";
+    public static final String VARIANT_NUM_COLUMN = "variantNum";
     static final String ID_COLUMN = "id";
     static final String VOLTAGE_LEVEL_ID_COLUMN = "voltageLevelId";
     static final String VOLTAGE_LEVEL_ID_1_COLUMN = "voltageLevelId1";
     static final String VOLTAGE_LEVEL_ID_2_COLUMN = "voltageLevelId2";
     static final String VOLTAGE_LEVEL_ID_3_COLUMN = "voltageLevelId3";
     static final String NAME_COLUMN = "name";
-    static final String EQUIPMENT_TYPE_COLUMN = "equipmentType";
+    public static final String EQUIPMENT_TYPE_COLUMN = "equipmentType";
     static final String REGULATING_EQUIPMENT_TYPE_COLUMN = "regulatingEquipmentType";
     static final String REGULATED_EQUIPMENT_TYPE_COLUMN = "regulatedEquipmentType";
-    static final String EQUIPMENT_ID_COLUMN = "equipmentId";
+    public static final String EQUIPMENT_ID_COLUMN = "equipmentId";
     static final String REGULATING_EQUIPMENT_ID = "regulatingEquipmentId";
     static final String INDEX_COLUMN = "index";
     static final String TAPCHANGER_TYPE_COLUMN = "tapChangerType";
     static final String ALPHA_COLUMN = "alpha";
-    static final String OPERATIONAL_LIMITS_GROUP_ID_COLUMN = "operationalLimitsGroupId";
-    static final String SELECTED_OPERATIONAL_LIMITS_GROUP_ID_COLUMN = "selectedOperationalLimitsGroupId";
+    static final String TEMPORARY_LIMITS_TABLE = "newtemporarylimits";
```

Comment:
avoid word "new" it becomes obsolete quickly

Reply:
Done